DOC.md

```diff
@@ -1776,6 +1776,189 @@ below) it will get run after the builtin callback.
 }
 ```
 
+## Treesitter-Postfix-Snippet
+
+Instead of triggering a postfix-snippet when some pattern matches in front of
+the trigger, it might be useful to match if some specific treesitter-nodes
+surround/are in front of the trigger.  
+While this functionality can also be implemented by a cusutom
+`resolveExpandParams`, this helper simplifies the common cases.  
+
+This matching of treesitter-nodes can be done either
+* by providing a query and the name of the capture that should be in front of
+  the trigger (in most cases, the complete match, but requiring specific nodes
+  before/after the matched node may be useful as well), or
+* by providing a function that manually walks the node-tree, and returns the
+  node in front of the trigger on success (for increased flexibility).
+
+A simple example, which surrounds the previous node's text preceeding the `.mv`
+with `std::move()` in cpp files, looks like:
+
+```lua
+treesitter_postfix({
+    trig = ".mv",
+    matchTSNode = {
+        query = [[
+            [
+              (call_expression)
+              (identifier)
+              (template_function)
+              (subscript_expression)
+              (field_expression)
+              (user_defined_literal)
+            ] @prefix
+        ]]
+        query_lang = "cpp"
+    },
+},{
+    f(function(_, parent)
+        return ("std::move(%s)"):format(parent.snippet.env.LS_TSMATCH)
+    end)
+})
```

Comment:
Seems that LS_TSMATCH is a `string[]`, can not call `replace` directly here.

Reply:
Ah, yes, ty!